### DISCUSSION TITLE
Request: Add Inter font

### DISCUSSION THREAD

**Discussion by mainrs** [ryanoasis/nerd-fonts#770] [Ideas]
[Inter](https://github.com/rsms/inter) is a font that focuses on UI. It would be great to see it patched as well. This would allow one to use the font in polybar for example :) 

It should be noted that Inter provides "advanced" features through OpenType. Not sure how that plays with nerd fonts. 

__Edit:__ I found an AUR script that does this: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=nerd-fonts-inter Looks that, for some reason, they had to patch font-patcher

**Comment by Finii** [comment]
The patch has already been applied to 'our' `font-patcher`; they just use the release version and we had no release in a long time now.

So you can not ingest AUR, or is the request obsolete?

OpenType Features are usually no problem.

**Reply by mainrs** [reply]
The request is not obsolete, I cannot ingest AUR; I run NixOS.

**Reply by Finii** [reply]
Created a closed Issue and linked it to the Add-Font-Metathread
